src/ui/qgsoptionsbase.ui

```diff
@@ -4888,6 +4888,46 @@ The bigger the number, the faster zooming with the mouse wheel will be.</string>
                     </property>
                    </widget>
                   </item>
+                  <item row="1" column="0">
+                   <widget class="QLabel" name="label_44">
+                    <property name="text">
+                     <string>Angle tolerance when tracing curves</string>
+                    </property>
+                   </widget>
+                  </item>
+                  <item row="2" column="0">
+                   <widget class="QLabel" name="label_70">
+                    <property name="text">
+                     <string>Distance tolerance when tracing curves</string>
+                    </property>
+                   </widget>
+                  </item>
+                  <item row="1" column="2">
+                   <widget class="QgsDoubleSpinBox" name="mTracingCustomAngleToleranceSpinBox">
+                    <property name="toolTip">
+                     <string>Parameter used by the convert to curves algorithm when tracing curves.</string>
```

Comment:
```suggestion
                     <string>Parameter used by the convert to curves algorithm when tracing curves.</string>
```

Could this be a tiny bit more explanatory? I don't know if you have a good wording for *how* it works, but already knowing what happens if it's lower or greater (more tolerance vs. stricter lower chance of unintentional detection of curves) would be nice.

Reply:
Other things to potentially mention:

 - rounding / floating point
 - depends on significant digits in the coordinate values

Reply:
Yes it's worth explaining, but wonder if the tooltip is the best place to do so ? Due to the complexity of the setting, it's hard to have anything useful in not too many lines...

Maybe we could improve the help text of the convert to curves algorithm, and refer to that instead ?

Reply:
That depends on a plugin being enabled and it's hard to locate.
I would not mind putting a lot of text onto the tooltip (and copy paste it to the alg), I think tooltips can even have HTML, so a short abstract and a longer explanation could be separated into paragraphs?

Reply:
Ok will try that